rich/style.py

```diff
@@ -740,7 +740,7 @@ def __add__(self, style: Optional["Style"]) -> "Style":
         new_style._link = style._link or self._link
         new_style._link_id = style._link_id or self._link_id
         new_style._hash = style._hash
-        new_style._null = self._null or style._null
+        new_style._null = style._null
```

Comment:
There's a guard about 10 lines above:

```python
if self._null:
    return style
```

So `self._null` cannot be truthy at this point.